src/settings/components/ProxyServerSettingsConfig/ProxyServerSettingsView/ProxyServerSettingsView.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { renderWithIntl, TestForm } from '@folio/stripes-erm-testing';
+import { Button, KeyValue } from '@folio/stripes-testing';
+
+import translationsProperties from '../../../../../test/helpers';
+import { input, meta } from './testResources';
+import ProxyServerSettingsView from './ProxyServerSettingsView';
+
+const onDelete = jest.fn();
+const onEdit = jest.fn();
+const onSubmit = jest.fn();
+
+describe('ProxyServerSettingsView', () => {
+  let renderComponent;
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <TestForm onSubmit={onSubmit}>
+        <ProxyServerSettingsView
+          input={input}
```

Comment:
We should never send input or meta props in directly unless there's a very good reason to.